tern/formats/generator.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import abc
+
+
+class GenerateReport(object):
```

Comment:
Either name this Generator because the file is called `generator.py` or call the file generate_report.py. It's for better readability.

Reply:
Done! Renamed `GenerateReport` to just `Generate` and inherited from ABCMetaclass.